File: app/utils/UUID.lua
local ____exports = {}
local function UUID()
    local uuids = {}
    local function get(key)
        if uuids[key] == nil then
            uuids[key] = 0
        end
        uuids[key] = uuids[key] + 1
        return (key .. "_") .. tostring(uuids[key])
    end
    return {get = get}
end
____exports.default = UUID()
return ____exports

File: src/utils/UUID.ts

function UUID() {
    const uuids: { [key: string]: number } = {};

    function get(key: string): string {
        if (uuids[key] == undefined) uuids[key] = 0;
        uuids[key] += 1;
        return key + "_" + uuids[key];
    }

    return {
        get
    };
}

export default UUID();